### PR TITLE
fix(console): add tooltips on documentation list

### PR DESCRIPTION
**Pull request by ytvnr**
## Issue

https://gravitee.atlassian.net/browse/APIM-3397

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-zoajluqdgp.chromatic.com)

### CODE REVIEW

File: gravitee-apim-console-webui/src/management/api/documentation-v4/documentation-pages-list/api-documentation-v4-pages-list.component.html [hunk 1]
```diff
@@ -157,6 +161,7 @@
             (click)="onDeletePage.emit(page)"
             [disabled]="page.generalConditions"
             aria-label="Delete page"
+            matTooltip="Delete page"
```
Comment (jourdiw) [inline gravitee-apim-console-webui/src/management/api/documentation-v4/documentation-pages-list/api-documentation-v4-pages-list.component.html]: Should have a specific tooltip to explain why trashcan is disabled --> `Cannot delete page used by active plans`

**Review by Okhelifi — APPROVED** [review]

File: gravitee-apim-console-webui/src/management/api/documentation-v4/documentation-pages-list/api-documentation-v4-pages-list.component.html (hunk 1 above)
Comment (ytvnr) [reply gravitee-apim-console-webui/src/management/api/documentation-v4/documentation-pages-list/api-documentation-v4-pages-list.component.html]: @jourdiw what about  `Cannot delete page used as general conditions`

Comment (jourdiw) [reply gravitee-apim-console-webui/src/management/api/documentation-v4/documentation-pages-list/api-documentation-v4-pages-list.component.html]: Yes, with `General Conditions` capitalized. Just make sure that the tooltip message doesn't go onto 2 lines... I had some issues with that for the Unpublish tooltip message when disabled.

Comment (ytvnr) [reply gravitee-apim-console-webui/src/management/api/documentation-v4/documentation-pages-list/api-documentation-v4-pages-list.component.html]: @jourdiw  
image

**Pull request merged**